test/language/statements/class/subclass/class-definition-null-proto-missing-return-override.js

```diff
@@ -38,4 +38,4 @@ class Foo extends null {
 
 var foo = new Foo();
```

Comment:
Should have mentioned in the PR description: the modified version of this test passes with a local version of V8 I have in which "extends null" has the newly-specced behavior.